README.md

```diff
@@ -329,14 +329,19 @@ cause the application to `abort()` on programming errors that would be logged
 and ignored at runtime.
 
 
-### Skipping the valgrind tests
+### Running tests with valgrind
 
-If you are trying to iterate quickly, you can temporarily skip the valgrind
-memory tests by running the test suite with:
 ```
-MMD_SKIP_VALGRIND=True ninja test
+meson test --suite=ci_valgrind \
+  --wrap="valgrind \
+          --leak-check=full \
+          --suppressions=/usr/share/glib-2.0/valgrind/glib.supp \
+          --suppressions=$SRCDIR/contrib/valgrind/libmodulemd-python.supp
```

Comment:
```suggestion
          --suppressions=../contrib/valgrind/libmodulemd-python.supp"
```

Reply:
I tweaked that section a little to make the path clearer.